WordPress/Classes/ViewRelated/Stats/Insights/StatsBaseCell.swift

```diff
@@ -1,7 +1,44 @@
 import UIKit
 
 class StatsBaseCell: UITableViewCell {
-    let headingLabel: UILabel = UILabel()
+
+    private let headingLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.preferredFont(forTextStyle: .headline)
+        return label
+    }()
+
+    private let showDetailsButton: UIButton = {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = true
+        button.addTarget(self, action: #selector(detailsButtonTapped), for: .touchUpInside)
+        button.titleLabel?.font = UIFont.preferredFont(forTextStyle: .callout)
+        button.tintColor = .secondaryLabel
+        button.setTitleColor(.secondaryLabel, for: .normal)
+        button.setImage(UIImage(named: "disclosure-chevron")?.withTintColor(UIColor(color: WPStyleGuide.greyLighten20())), for: .normal)
```

Comment:
I think we can probably just use `.gridicon(.chevronRight)`

Reply:
Done!